controllers/controller.js

```diff
@@ -23,15 +23,22 @@ function retrieveArticleById(req, res, next) {
 
 function retrieveArticles(req, res, next) {
     getArticlesSorted()
-    .then((result) => {
-        const articlesToSend = result.map((article) => {
-            delete article.body;
-            return article;
-        })
-        res.status(200).send({ articles: articlesToSend });
+    .then((sortedArticles) => {
+        res.status(200).send(sortedArticles);
     }).catch((error) => {
         next(error);
     });
 }
 
-module.exports = { retrieveTopics, retrieveArticleById, retrieveArticles };
\ No newline at end of file
+function retrieveCommentById(req, res, next) {
+    const {id} = req.params;
+    // console.log(req.query);
+    getCommentByArticle(id)
+    .then((comments) => {
+        res.status(200).send(comments);
```

Comment:
We should always send back our response as an object with an appropriate key, like {comments: comments} or just {comments} using the shorthand.

Reply:
This does mean you'll need to update the test so when you do this ensure you update the test first 👍